src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -56,6 +59,7 @@ namespace plugins
   ///     * \<near\> : Camera's near clipping plane distance, defaults to 0.01
   ///     * \<far\> : Camera's far clipping plane distance, defaults to 1000.0
   /// * \<sky\> : If present, sky is enabled.
+  /// * \<graphics_api\> : Optional graphics API name, defaults to 'opengl'.
```

Comment:
I would list other valid parameters here (such as `metal`)

Reply:
Updated in [6749a92](https://github.com/ignitionrobotics/ign-gui/pull/323/commits/6749a92b9987f56246696b6f1ee0d4cdbcb3039c)